content/docs/conditional-rendering.md

```diff
@@ -35,11 +35,9 @@ function Greeting(props) {
   return <GuestGreeting />;
 }
 
-ReactDOM.render(
-  // Try changing to isLoggedIn={true}:
-  <Greeting isLoggedIn={false} />,
-  document.getElementById('root')
-);
+ReactDOM
```

Comment:
Please write it in two lines:

```js
const root = ...
root.render(...)
```

thanks!

Reply:
Hello @gaearon, good morning! 

I have reviewed your request and made changes as requested. You could review the changes again. 

Thank you!